### PR TITLE
fix: update deps, reconfigure tests to reflect most recent polkadot-js changes

### PR DESCRIPTION
**Pull request by TarikGul**
This PR updates the deps for polkadot-js, and substrate/dev

I also reconfigured tests in order to reflect the most recent internal changes to polkadot-js [here](https://github.com/polkadot-js/api/commit/400e14d6c94613e3b013f207607e4c16c61d9b4b#diff-4057209e51321417a7ba8384eeaa7221603262b495ae1f975753c129de848f4eR24)

### CODE REVIEW

File: src/test-helpers/constants.ts [hunk 1]
```diff
@@ -22,3 +22,9 @@ export const MAX_U8 = '255';
 
 export const MAX_I8 = '127';
 export const MIN_I8 = '-128';
+
+export const H512_U64 =
```
Comment (niklasad1) [inline src/test-helpers/constants.ts:26]:
```suggestion
export const MAX_H512 =
```

Comment (TarikGul) [reply src/test-helpers/constants.ts]: I like the name suggestion!
Just changed them all to reflect this.

**Review by niklasad1 — APPROVED** [review]

File: src/sanitize/mockData.ts [hunk 2]
```diff
@@ -30,7 +30,7 @@ export const PRE_SANITIZED_BALANCE_LOCK = kusamaRegistry.createType(
 	'Vec<BalanceLock>',
 	[
 		{
-			id: 'LockId',
+			id: '00000000',
```
Comment (dvdplm) [inline src/sanitize/mockData.ts:33]: What happened here?

**Review by dvdplm — APPROVED** [review]
lgtm.

I think in general that we should try to be explicit about any numbers that appear in tests. If they are just random numbers or have a meaning, it'd be helpful to see a short comment as a hint of "don't bother wondering where this came from".

File: src/sanitize/mockData.ts (hunk 2 above)
Comment (TarikGul) [reply src/sanitize/mockData.ts:33]: It expected the id to be of 8 bytes but instead `lockId` is 6 bytes

**Pull request merged**